tools/DebianPackageTool/package_tool

```diff
@@ -152,6 +152,9 @@ package_all(){
     package_absolute_placement
     package_samples
     package_docs
+    if [[ $PACKAGE_NAME == "dotnet-dev-"* ]]; then
+        package_debian
```

Comment:
FWIW, I think the name of this function is confusing.  I would not expect the guard given the general name (e.g. wouldn't I always want to package postinst if it was present?